cms/static/js/views/video/transcripts/file_uploader.js

```diff
@@ -18,7 +18,9 @@ function($, Backbone, _, Utils) {
         uploadTpl: '#file-upload',
 
         initialize: function () {
-            _.bindAll(this);
+            _.bindAll(this,
```

Comment:
Note to reviewers: must now pass the method names to bindAll (can't bind "all"!).